201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,202 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of existing VM Scale Set"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Number of desired VM instances"
+            }
+        },
+        "keyVaultName": {
+          "type": "string",
+          "metadata": {
+            "description": "Name of the KeyVault to place the volume encryption key"
+          }
+        },
+        "keyVaultResourceGroup": {
+          "type": "string",
+          "metadata": {
+            "description": "Resource group of the KeyVault"
+          }
+        },
+        "keyEncryptionKeyURL": {
+          "type": "string",
+          "defaultValue": "",
+          "metadata": {
+            "description": "URL of the KeyEncryptionKey used to encrypt the volume encryption key"
+          }
+        }
+    },
+    "variables": {
+        "namingInfix": "existingvmss",
+        "virtualNetworkName": "[concat(variables('namingInfix'), 'vnet')]",
+        "publicIPAddressName": "[concat(variables('namingInfix'), 'pip')]",
+        "subnetName": "[concat(variables('namingInfix'), 'subnet')]",
+        "loadBalancerName": "[concat(variables('namingInfix'), 'lb')]",
+        "natPoolName": "[concat(variables('namingInfix'), 'natpool')]",
+        "bePoolName": "[concat(variables('namingInfix'), 'bepool')]",
+        "keyVaultResourceID": "[resourceId(parameters('keyVaultResourceGroup'), 'Microsoft.KeyVault/vaults/', parameters('keyVaultName'))]"
+    },
+    "resources": [{
+            "type": "Microsoft.Network/virtualNetworks",
+            "name": "[variables('virtualNetworkName')]",
+            "location": "[resourceGroup().location]",
```

Comment:
Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

Reply:
Addressed these comments in commit# 9f41323.